README.md

```diff
@@ -1,13 +1,13 @@
-# Nyaaya V2 Prototype
+# Nyaaya CMS
 
-simple node app to test out features and models for Nyaaya V2
+Keystone.js based CMS for handling the backend of nyaaya-web. 
```

Comment:
link to keystone.js here and also mention which version we're using

Reply:
resolved